development/compiling/compiling_for_osx.rst

```diff
@@ -17,7 +17,7 @@ For compiling under macOS, the following is required:
 - *Optional* - `yasm <https://yasm.tortall.net/>`_ (for WebM SIMD optimizations).
 
 If you are building the ``master`` branch:
-- Download and install the `Vulkan SDK for macOS <https://vulkan.lunarg.com/sdk/home>`__.
+- Download and install (install SDK only if you want to use standalone executable) the `Vulkan SDK for macOS <https://vulkan.lunarg.com/sdk/home>`__.
```

Comment:
This addition is not clear to me, and the insert makes the sentence a bit weird. I'd suggest rephrasing.

Reply:
System-wide (to `/usr/local`) install script is messy, can cause conflicts, and pointless if you want to use `.app` bundle, but can be convenient for stand-alone executable.

But since current SDK documentation "Install the SDK" page say "mount a disk image file and copy the contents into a desired directory", this addition is probably excessive. I'll remove it completely.